security-release-process.md

```diff
@@ -123,17 +123,17 @@ Their rotation will involve the following:
 
 #### SIG Release Roles
 
-Included on the [private Release Managers list](https://groups.google.com/a/kubernetes.io/forum/#!forum/release-managers-private)
+Included on the [private Release Managers list](https://groups.google.com/a/kubernetes.io/forum/#!forum/security-release-team)
 are the following members:
 
-- [Release Managers](https://git.k8s.io/sig-release/release-managers.md#release-managers)
+- [Release Managers](https://git.k8s.io/website/content/en/releases/release-managers.md)
```

Comment:
If we're linking to the website content, we might as well link to the actual website:
```suggestion
- [Release Managers](https://kubernetes.io/releases/release-managers/)
```